test_conformance/compiler/test_compile.cpp

```diff
@@ -3933,3 +3941,96 @@ REGISTER_TEST(compile_and_link_status_options_log)
 
     return 0;
 }
+
+REGISTER_TEST(multiple_build_program)
+{
+    int error = CL_SUCCESS;
+    size_t threads[1] = { 1 };
```

Comment:
I don't think `threads` needs to be an array type for a single element. Also the term `threads` is a bit overloaded with host side threading, would prefer a name like `total_work_items` or `global_size`, etc